src/test/java/org/folio/des/scheduling/quartz/QuartzExportJobSchedulerTest.java

```diff
@@ -71,8 +72,9 @@ void testSchedule() throws SchedulerException {
     assertEquals(1, schedulerListener.getJobsAddedCount());
     assertEquals(1, schedulerListener.getJobsScheduledCount());
 
-    await().pollDelay(1, TimeUnit.SECONDS).timeout(10, TimeUnit.SECONDS).untilAsserted(
-      () -> assertEquals(1, jobListener.getJobsExecutedCount()));
+    // TODO - Should be reworked because this code leads to the test failure due to unstable result
```

Comment:
Consider changing this validation. It fails from time to time.

Reply:
in our recent prs where we introduced quartz - we did not see any issues with this test